tests/MockTest.php

```diff
@@ -125,6 +131,82 @@ public function testDisableAll()
         Mock::disableAll();
 
         $this->assertNotEquals(1234, time());
-        $this->assertEquals(1, min(1, 2));
+        $this->assertEquals(1, min(array(1, 2)));
```

Comment:
The function signature changed between 5.5 and 5.6. In PHP5.5 the Reflection-API gave back 3 parameters for the ```min```-Function (```$param1```, ```$param2``` and ```...```) whereas in 5.6 that changed to 1 parameter (```array $array```) - In 5.6 the method also accepts the 5.5 way and vice versa. But as I used the Reflections function signature to get the parameters I had to change that test to work in PHP 5.6. 

I currently do not have a clue on how to change that while preserving the possibility to pass parameters by reference.